### DISCUSSION TITLE
reComputer Industrial J20 Hardware and Interfaces Usage | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1350] [Q&A]
# reComputer Industrial J20 Hardware and Interfaces Usage | Seeed Studio Wiki

Hardware and Interfaces Usage for reComputer Industrial J20 Series

https://wiki.seeedstudio.com/reComputer_Industrial_J20_Hardware_Interfaces_Usage/

**Comment by almarion** [comment]
More details on this for use on  reComputer Industrial J3011 POE Camera on PSE Port (validation example - camera model & software  would be helpful)

"The leftmost connector is directy connected to the Jetson module and is able to provide PoE functionality with PSE 802.3 af, 15W specification. This means you can connect a PoE IP camera or any other PoE device to this port to provide power to the connected device".

**Comment by NiemalsSebi** [comment]
As of Jetpack 6.0 the guide to use the GPIO is deprecated. gpiod hast to be used. However on my Industrial Jetson Orin Nano The table is not the same as shown.

**Reply by almarion** [reply]
Hi,
I have a reComputer Industrial J3011 and just wanted to ask you if you can confirm that a POE IP Camera connected to the PSE Port works for you and if so can you describe in detail the camera model, the Network Settings and the verification software used? I am using Jetpack6.  The camera works fine when using a POE injector and connected via the normal LAN.  Also the camera appears to be powered by POE (i.e. LEDS on camera illuminate) when connected to the PSE Port although it still isn't accessible.  
Grateful for your advice,
Alan

**Reply by NiemalsSebi** [reply]
Hello! Did you check if the port has an appropiate IP address?

**Reply by almarion** [reply]
Thanks for your reply.
In summary, my camera ONLY works with a separate POE injector and on the LAN via “Microchip Ethernet” port. The “Realtek Ethernet” PSE port is configured for Manual IP (192.168.0.117) & (255.255.255.0) mask. This is also the IP address configured for the camera.

**Reply by NiemalsSebi** [reply]
Hello again!
Well, so I tested with my Windows PC First and it works fine when my Lan IP is on the same Subnet but Address is DIFFERENT from Camera (.65 Port, .64 Camera in /24 Subnet).
With this knowledge I tested it on the Orin, with and without the POE-Port. However I could not access or ping the camera (.64). It did not matter if I used Realtek or Microchip. Since I have to downgrade anyways I will give you an update on JP 5.

**Comment by snowbotixadmin** [comment]
Drivers for SONY ISX031 CSI MIPI?

**Reply by bleaaach** [reply]
Hi, we apologize, but we do not provide the driver for this camera. However, you can compile it yourself by following our tutorial.
 https://wiki.seeedstudio.com/how_to_build_the_ko_module_for_seeed_jetson/
